Node2Code_Engine/Create/LoopStatement.cs

```diff
@@ -0,0 +1,31 @@
+﻿using BH.oM.Node2Code;
+using BH.oM.Programming;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BH.Engine.Node2Code
+{
+    public static partial class Create
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        public static StatementInfo LoopStatement(INode node, List<int> depthDiffs)
```

Comment:
Documentation? Even though this is empty and would probably be better off being hidden?